src/common/common.cpp

```diff
@@ -726,6 +726,71 @@ bool handleViKey(QKeyEvent *event, QObject *eventReceiver)
     return true;
 }
 
+bool handleEmacsKey(QKeyEvent *event, QObject *eventReceiver)
+{
+    int key = event->key();
+    Qt::KeyboardModifiers mods = event->modifiers();
+
+    switch ( key ) {
+    case Qt::Key_V:
+        if (mods & Qt::ControlModifier) {
+            key = Qt::Key_PageDown;
+            mods = mods & ~Qt::ControlModifier;
+        } else if (mods & Qt::AltModifier) {
+            key = Qt::Key_PageUp;
+            mods = mods & ~Qt::AltModifier;
+        } else {
+        return false;
+    }
```

Comment:
To keep the code structure consistent:

```suggestion
            mods = mods & ~Qt::ControlModifier;
            break;
        }
        if (mods & Qt::AltModifier) {
            key = Qt::Key_PageUp;
            mods = mods & ~Qt::AltModifier;
            break;
        }
        return false;
```